templates/taxi/driver_list.html

```diff
@@ -4,10 +4,13 @@
   <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
-      {% for driver in driver_list%}
+      {% for driver in driver_list %}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          <a href="{% url 'taxi:driver-detail' pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver == current_user %}
+            (Me)
+          {% endif %}
```

Comment:
The variable `current_user` is used here, but it is not defined in the context passed to the template. Ensure that the view providing this template includes `current_user` in its context.